config/initializers/statsd.rb

```diff
@@ -26,21 +26,43 @@ class << Samson
 end
 
 ActiveSupport::Notifications.subscribe("job_queue.samson") do |*, payload|
-  payload.each { |key, value| Samson.statsd.gauge "job.#{key}", value }
+  # deploys running/queued
```

Comment:
`[[:deploys, true], [:jobs, false]].each` would be nice here also to make sure both are tagged for simpler display logic (true/false vs true/"")